src/SystemCommands-RefactoringSupport/RBPushDownPreview.class.st

```diff
@@ -0,0 +1,121 @@
+"
+I am a preview of push down method refactoring, I can select the methods of class to push down
+"
+Class {
+	#name : #RBPushDownPreview,
```

Comment:
I think it would be better to use full name: RBSelectorsPushDownPreview because there is similar push down command for variables and we can implement another preview for it.